frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -45,16 +43,31 @@ export const CommandPalette: FC = () => {
     return () => document.removeEventListener('keydown', down)
   }, [])
 
+  // Select option by pressing [Enter] key (with/without ⌘ key)
+  useEffect(() => {
+    const down = (event: KeyboardEvent) => {
+      if (!open || !tableName) return
+
+      if (event.key === 'Enter') {
+        if (event.metaKey || event.ctrlKey) {
+          window.open(`?active=${tableName}`)
+        } else {
+          goToERD(tableName)
+        }
+      }
```

Comment:
For mouse use, it behaves as a normal link element while ⌘ (or ctrl) being hold, otherwise it goes to ERD which is the same behaviour with the previous version.